package-system/SPIRVCross/build_config.json

```diff
@@ -1,32 +1,17 @@
 {
-   "git_url":"https://github.com/o3de/SPIRV-Cross.git",
-   "git_tag":"release-2021.04.29",
+   "git_url":"https://github.com/KhronosGroup/SPIRV-Cross.git",
+   "git_tag":"vulkan-sdk-1.3.275.0",
    "package_name":"SPIRVCross",
-   "package_version":"2021.04.29-rev1",
-   "package_url":"https://github.com/03de/SPIRV-Cross",
+   "package_version":"1.3.275.0-rev1",
+   "package_url":"https://github.com/KhronosGroup/SPIRV-Cross",
    "package_license":"Apache-2.0",
    "package_license_file":"LICENSE",
    "cmake_find_target":"FindSPIRVCross.cmake",
    "Platforms":{
       "Windows":{
          "Windows":{
             "cmake_find_template":"FindSPIRVCross.cmake.Windows",
-            "cmake_generate_args_debug":[
-               "-G",
-               "\"Visual Studio 16 2019\"",
-               "-A",
-               "x64",
-               "-T",
-               "host=x64",
-               "-DSPIRV_CROSS_CLI=ON",
-               "-DSPIRV_CROSS_SHARED=OFF",
-               "-DCMAKE_INSTALL_LIBDIR=\"lib/Debug\"",
-               "-DCMAKE_INSTALL_BINDIR=\"bin/Debug\"",
-               "-DCMAKE_BUILD_TYPE=Debug"
-            ],
```

Comment:
Why was the entire `cmake_generate_args_debug` removed? Its causing the `pull_and_build_from_git` script to fail. You should only need to remove the `-G`, `-A`, `-T`, and `host=x64` arguments

Reply:
If the intention is to only build on the release target, then a `--build_configs release` argument should be added to each `package_build_list_host_*.json`command. 

`pull_and_build_from_git.py` defaults to the release and debug target if no argument is given: https://github.com/o3de/3p-package-source/blob/main/Scripts/extras/pull_and_build_from_git.py#L279C11-L279C103

Reply:
I brought the debug configs back to allow the packages to be built.